couchbase/error_context/analytics.hxx

```diff
@@ -39,6 +39,8 @@ struct analytics {
     std::string path{};
     std::uint32_t http_status{};
     std::string http_body{};
+    std::string hostname{};
```

Comment:
It seems like it would be better to have the hostname/port as a single string, as it is specified in the last_dispatched_to and served_by_node fields since error_context's are also user-facing.